howtos/release-procedure.md

```diff
@@ -63,34 +63,50 @@ To create the new versions:
    ];
    ```
 
-   - Order of the items in the array is important. Add the new mapping in the first position of the array, because the first item is treated as the "current" version mapping.
+   Caution: Order of the items in the array is important. Add the new mapping in the first position of the array, because the first item is treated as the "current" version mapping.
```

Comment:
Non-blocking, but did you intentionally remove the `-`?

Reply:
I did. This made more sense as an "admonition" style paragraph than a bullet point.